.env.test

```diff
@@ -51,4 +51,17 @@ REACT_APP_GA_API_KEY=""
 # - used to fetch token balances on wallet
 # - you can pick up a Covalent key at https://www.covalenthq.com/
 REACT_APP_COVALENT="MOCK_API_KEY"
+
+# Zapper
+# Public key from Zapper, ONLY works on MAINNET
+# Zapper keys aren't needed unless you are trying to zap.
+# Zaps aren't available except on mainnet
+REACT_APP_ZAPPER_API="96e0cc51-a62e-42ca-acee-910ea7d2a241"
+REACT_APP_ZAPPER_POOL="0x04f2694c8fcee23e8fd0dfea1d4f5bb8c352111f"
+
+# Olympus Give
+# These 2 settings should be toggled to "true" if you want to use Olympus Give features
+REACT_APP_GIVE_ENABLED="true"
+REACT_APP_MOCK_SOHM_ENABLED="true"
```

Comment:
I would also leave this as false. REACT_APP_MOCK_SOHM_ENABLED is only needed for specific cases when testing the give functionality. It will result in a developer's OHM "disappearing".

Reply:
OK, this is a good point to discuss then. My thinking with setting these to true was to enable tests to cover Give functionality. What is the correct way to do this?

Reply:
At this stage it doesn't affect tests, so I would leave line 65 out.